tests/integration/test_authd/test_authd.py

```diff
@@ -92,9 +93,13 @@
 def set_up_groups(request):
     groups = request.param.get('groups', [])
     for group in groups:
+        logging_message('FunctionLog', 'V', f"set_up_groups - Adding group {group}")
```

Comment:
This log should be `VV` since it shows information about what the function is doing, but no errors or warnings. We should check this in every logging message of this file.